### ISSUE TITLE
Address a11y issues from axe, WAVE, NVDA

### ISSUE THREAD

**Issue by candu** [CityofToronto/bdit_flashcrow#327]
**What Happened**
axe and WAVE identified several small issues with accessibility, mostly around non-visual cues and color contrast.

In addition, NVDA surfaced some areas where available text is insufficient (e.g. autocomplete dropdowns).

**What Should Happen**
axe and WAVE should show no errors / warnings, with the exception of those requiring manual review (e.g. contrast of controls over map, where contrast cannot be determined due to `<canvas>` element).

MOVE should be navigable by keyboard, and NVDA should announce meaningful and understandable text for interactive elements.

**To Reproduce**
For axe / WAVE issues:
1. Install axe / WAVE extensions
2. Open MOVE
3. Run axe / WAVE checks on each page within MOVE

For NVDA issues:
1. Install NVDA
2. Open MOVE
3. Use NVDA / keyboard to navigate MOVE

**Additional Notes**
Map accessibility will require a lot of manual work, and can be addressed slightly later (e.g. once we complete the map redesign).
